media/com_associations/js/admin-autoassoc-modal.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Please use ES6 here:
```js
(() => {
  'use strict';

  const onClick = (event) => {
    event.preventDefault();
    const checkedBoxes = [].slice.call(document.querySelectorAll("td.row-selected input[type='checkbox']"));
    const assocLanguages = document.querySelector("input[name='assocLanguages']");
    const languageIds = [];

    if (checkedBoxes.lenght) {
      checkedBoxes.forEach((box) => {
        languageIds.push(box.value);
      });
    }

    if (languageIds.length) {
      languageIds.forEach((languageId, index) => {
        if (index === 0) {
          assocLanguages.value = languageId;
        } else {
          assocLanguages.value += (`:${languageId}`);
        }
      });
    }

    Joomla.submitform('autoassoc.autocreate'); 
  };

  const onBoot = () => {
    const applyBtn = document.getElementById('applyBtn');

    if (applyBtn) {
      applyBtn.addEventListener('click', onClick);
    }
    
    // Cleanup
    document.removeEventListener('DOMContentLoaded', onBoot, true);
  };
  
  document.addEventListener('DOMContentLoaded', onBoot, true);
})();
```

By the way there were some errors in the script

Reply:
Thank you! @dgrammatiko @infograf768